docs/user_guide.md

```diff
@@ -556,76 +518,174 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+   * **Grey:** **Unknown** level of vulnerabilities
+1. Hover over the number of fixable vulnerabilities to see a summary of the vulnerability report. 
+   ![Vulnerability summary](img/vulnerability-summary.png)
+1. Click on the tag name to see a detailed vulnerability report. 
+  ![Vulnerability report](img/tag_detail.png)
+  
+   In addition to information about the tag, all of the vulnerabilities found in the last scan are listed. You can order or filter the list by the different columns.
 
-Move the cursor over the bar, a tooltip with summary report will be displayed. Besides showing the total number of features with vulnerabilities and the total number of features in the scanned image tag, the report also lists the counts of features with vulnerabilities of different severity levels. The completion time of the last analysis process is shown at the bottom of the tooltip.
-![browse project](img/summary_tooltip.png)
+### Scan All Images
 
-Click on the tag name link, the detail page will be opened. Besides the information about the tag, all the vulnerabilities found in the last analysis process will be listed with the related information. You can order or filter the list by columns.
-![browse project](img/tag_detail.png)
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
+1. Expand **Administration**, and select **Interrogation Services**. 
+1. Select the **Vulnerability** tab and click **Scan Now** to scan all of the images in all projects.
+   ![Scan all images](img/scan_all.png)
 
-**NOTES: You can initiate the vulnerability analysis for a tag at anytime you want as long as the status is not 'Queued' or 'Scanning'.** 
+The scanning process runs multiple concurrent asynchronous tasks. The order in which the tasks are run and the results returned is not guaranteed.
 
-**Scanning all images**
+Scanning requires intensive resource consumption. To avoid frequently triggering scans too frequently, scans can be only triggered once in a defined period. If scanning is unavailable, the next available time is displayed next to the **Scan Now** button.
 
-In the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**, click on the **'SCAN NOW'** button to start the analysis process for all the existing images. 
+### Schedule Scans
 
-**NOTES: The scanning process is executed via multiple concurrent asynchronous tasks. There is no guarantee on the order of scanning or the returned results.** 
-![browse project](img/scan_all.png)
+You can set policies to control when vulnerability scanning should run.
 
-To avoid frequently triggering the resource intensive scanning process, the availability of the button is restricted. It can be only triggered once in a predefined period. The next available time will be displayed besides the button.
-![browse project](img/scan_all2.png)
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
```

Comment:
@xaleeks we should settle on the terminology, and then I'll make it consistent throughout all of the docs. The options are:

- Harbor administrator
- System administrator
- Harbor system administrator
- Or maybe just "administrator", although this might be confused with project administrator.

Reply:
for now, maybe system admin or harbor system admin would be pretty clear

Reply:
I vote for Harbor system administrator. I did a global change in https://github.com/goharbor/harbor/pull/10029/commits/858acd32a241ee97b9e55bb8face771b7db83789.

Reply:
sounds good to me